modules/ivc_champva/app/jobs/ivc_champva/missing_form_status_job.rb

```diff
@@ -40,6 +40,10 @@ def perform
       Rails.logger.error e.backtrace.join("\n")
     end
 
+    def fetch_forms_by_uuid(form_uuid)
+      @fetch_forms_by_uuid ||= IvcChampvaForm.where(form_uuid:)
+    end
```

Comment:
This method is used down below to update the `email_sent` flag once emails have successfully fired off.